feat: Add Svelte SDK docs (#5413)
getsentry/sentry-docs: src/includes/sourcemaps/upload/primer/javascript.svelte.mdx

> To upload your Svelte project's sourcemaps to Sentry, you currently have two options:

Reviewer: ```suggestion
To upload your Svelte project's source maps to Sentry, you currently have two options:
```

> - Upload sourcemaps manually using [Sentry-CLI](./cli/). Take a look at this [bash script](https://github.com/getsentry/sentry-javascript/tree/master/packages/svelte#sourcemaps-and-releases) as an example how to configure the CLI for a typical Svelte project.

Reviewer: ```suggestion
- Upload source maps manually using [Sentry-CLI](./cli/). Take a look at this [bash script](https://github.com/getsentry/sentry-javascript/tree/master/packages/svelte#sourcemaps-and-releases) as an example of how to configure the CLI for a typical Svelte project.
```

> - If you are using Vite, you can use the [unofficial Sentry Vite plugin](https://github.com/ikenfin/vite-plugin-sentry) to set releases and upload sourcemaps. **Please note that this plugin is not maintained by Sentry and we do not offer support for it.**

Reviewer: ```suggestion
- Use the [unofficial Sentry Vite plugin](https://github.com/ikenfin/vite-plugin-sentry) to set releases and upload source maps. **Please note that this plugin is not maintained by Sentry and we do not offer support for it.**
```
